### ISSUE TITLE
Installer Issue on Multiboot drive

### ISSUE THREAD

**Issue by zaivala** [feren-OS/Bug-Reporting-Center#25]
**Package Name or Section of Feren OS where this bug resides**
Where does this bug happen? (e.g.: feren-themer-cinnamon) ___

**Describe the bug**

**Steps to Reproduce**
1. 

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by zaivala** [comment]
Odd how the problem was sent but all my report was deleted.

The installer, which appears to be Calamares, finds a multiboot system and I select which partition I want feren installed on instead of what is already there. 
As with OpenMandriva 3 before I reported the bug, it fails to do so, and reports the failure. So I have to go back into the installer. and start over. They fixed the bug before OpenMandriva 4 alpha.

In ferenOS, however, you got it a step closer to correct. It actually formats the partition to be installed on. It then tries to define the parition, however, as a new partition of the same size as the partition which is already (and still) there, and install to that imaginary partition.

When I went back in OM3, no amount of times I repeated the problem did it actually format; I got the error at the end of the imaginary format. In feren OS, however, the first time does actually format the partition, and the second time allows it to be installed.

**Comment by feren** [comment]
Definitely odd, but at the same time I'm not fully understanding the issue here... Heck this might even potentially be an upstream Calamares issue in the way you're describing it... What's your Calamares version? (Calamares Welcome Page > About)

**Comment by zaivala** [comment]
I have no idea what version of Calamares it is using, as it's not even identified as being Calamares, I just thought I recognized it from other repo installs... but I just downloaded the Feren OS installation the day or the day before I installed it.

I have been having a few other problems with Feren OS this install. I'm beginning to think I got some bad files or need to run the checksum... I've installed Feren OS on a few occasions in the past, and this is the first time I've had problems with it.
